src/ocamlorg_frontend/pages/tutorial.eml

```diff
@@ -24,9 +24,17 @@ let of_tutorial_section (s: Data.Tutorial.Section.t) =
   | Platform -> Learn_layout.Platform
   | Guides -> Learn_layout.Guides
 
+let render_related_exercises exercises =
+  let exercise_html = exercises |> List.map (fun exercise ->
```

Comment:
```suggestion
  let exercise_html = exercises |> List.map (fun (exercise : Data.Exercise.t) ->
```

I think this is one of the few situations in OCaml where we have to put a type annotation on the anonymous function. From what I've seen, type inference on anonymous functions seems to be different compared to type inference on functions defined via `let` binding.